cc_plugin_ncei/ncei_base.py

```diff
@@ -557,50 +541,254 @@ def check_global_attributes(self, dataset):
+            'creator_type',
+            'creator_institution',
+            'publisher_type',
+            'publisher_institution',
+            'program',
+            'contributor_name',
+            'contributor_role',
+            'geospatial_lat_units',
+            'geospatial_lon_units',
+            'geospatial_vertical_units',
+            'product_version',
+            'keywords_vocabulary',
+            'platform_vocabulary',
+            'instrument',
+            'instrument_vocabulary',
+            'metadata_link',
+            'references'
         ]
 
-        recommended_ctx = TestCtx(BaseCheck.MEDIUM, 'Recommended global attributes')
-        for attr in should_exist:
-            recommended_ctx.assert_true(getattr(dataset, attr, '') != '', '{} should exist and not be empty.'.format(attr))
+    def check_base_required_attributes(self, dataset):
+        '''
+        Check the global required and highly recommended attributes for 2.0 templates. These go an extra step besides
+        just checking that they exist.
 
-        # Do any of the variables define platform ?
-        variable_defined_platform = any((hasattr(var, 'platform') for var in dataset.variables))
-        if not variable_defined_platform:
-            platform_name = getattr(dataset, 'platform', '')
-            recommended_ctx.assert_true(platform_name and platform_name in dataset.variables, 'platform should exist and point to a variable.')
+        :param netCDF4.Dataset dataset: An open netCDF dataset
+
+        :Conventions = "CF-1.6, ACDD-1.3" ; //............................... REQUIRED - Always try to use latest value. (CF)
+        :featureType = "timeSeries" ; //..................................... REQUIRED - CF attribute for identifying the featureType.
+        :cdm_data_type = "Station" ; //...................................... REQUIRED (ACDD)
+        :ncei_template_version = "NCEI_NetCDF_TimeSeries_Orthogonal_Template_v1.1" ; //....... REQUIRED (NCEI)
+        :title = "" ; //............................................... HIGHLY RECOMMENDED - Provide a useful title for the data in the file. (ACDD)
+        :summary = "" ; //............................................. HIGHLY RECOMMENDED - Provide a useful summary or abstract for the data in the file. (ACDD)
+        :keywords = "" ; //............................................ HIGHLY RECOMMENDED - A comma separated list of keywords coming from the keywords_vocabulary. (ACDD)
+        :Conventions = "CF-1.6, ACDD-1.3" ; //......................... HIGHLY RECOMMENDED    - A comma separated list of the conventions being followed. Always try to use latest version. (CF/ACDD)
+        '''
+        test_ctx = TestCtx(BaseCheck.HIGH, 'Required global attributes')
+
+        conventions = getattr(dataset, 'Conventions', '')
+        feature_type = getattr(dataset, 'featureType', '')
+
+        # Define conventions
+        accepted_conventions = ['CF-1.6, ACDD-1.3', 'CF-1.6,ACDD-1.3']
```

Comment:
I don't think there's any doctrine on the order in which the conventions are listed or how much white space is allowed between delimiters. Could we do something like this instead?

```python
accepted_conventions = ['CF-1.6', 'ACDD-1.3']
dataset_conventions = conventions.replace(' ', '').split(',')
for accepted_convention in accepted_conventions:
    if accepted_convention not in dataset_conventions:
      result_fail
else:
    result_pass
```

Or something of the like, that way the check becomes agnostic to order and if someone mistakenly puts in like 30 spaces, which I've seen due to copy paste errors.